v1/tarball/testdata/README.md

```diff
@@ -0,0 +1,48 @@
+# Test Data
+
+The files in this directory are test data for tarball construction.
+
```

Comment:
Shouldn't be bad to get these generated.  I hate checking in binaries :p

Reply:
I can live with checking them in for things like malformed data or to exhibit behaviors that appear in the wild but rules_docker can't produce.  That's generally the line I've lived by. :)